kubernetes-csi-external-attacher-4.3.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+  name: kubernetes-csi-external-attacher-4.3
+  version: 4.3.0
+  epoch: 0
+  description: Sidecar container that watches Kubernetes VolumeAttachment objects and triggers ControllerPublish/Unpublish against a CSI endpoint
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    provides:
+      - kubernetes-csi-external-attacher=4.3.999
```

Comment:
I've been using `${{package.full-version}}` and making sure the epoch is above any previously built versions.

Reply:
Done, latest was 5, https://github.com/wolfi-dev/os/commit/f16857c460d5cbd2b5bdc7e30c67b13e7cb1a8a2